napari/_vispy/text_utils.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+
+def update_text(layer, text_node):
```

Comment:
I'd probably prefer this function to just take the `layer._text`, `layer._indices_view` , and `layer.dims.ndisplay` variables, it just feels a little safer, more modular, so function signature would be 

```python
def update_text(text, coordinates, ndisplay, text_node):
```
Does that sound reasonable?

Reply:
Yeah, I think that's totally reasonable!

Reply:
I definitely agree that we should limit passing the full layer model around as much as possible.

Reply:
Done!